deploy/roles/iptables/tasks/main.yml

```diff
@@ -6,7 +6,7 @@
 
 ---
 - include: rhel_7.yml
-  when: "{{ ansible_distribution_major_version }} == 7"
+  when: rh7_upgrade
```

Comment:
@lel99999 I think we want to stick with the original conditional statement, but add a predicate to check if we are in Vagrant or not as well

Reply:
Restored to original.  Conditional ENV == "DEV.local"  to be used for Vagrant